pyproject.toml

```diff
@@ -11,6 +11,43 @@ authors = [
 	{ name = "Jeremie Kalfon"}]
 readme = "README.md"
 dynamic = ["classifiers", "keywords", "license", "scripts", "version"]
+dependencies = [
+  "av",
+  "bokeh",
+  "coverage",
+  "cython",
+  "h5py",
+  "holoviews",
+  "ipykernel",
+  "ipython",
+  "ipyparallel",
+  "ipywidgets",
+  "keras",
+  "matplotlib",
+  "mypy",
+  "numpy",
+  "numpydoc",
+  "opencv-python",
+  "peakutils",
+  "pims",
+  "psutil",
+  "pynwb",
+  "scikit-image",
+  "scikit-learn",
+  "scipy",
+  "tensorflow",
+  "tifffile",
+  "tqdm",
+  "yapf",
+  "zarr"
+]
+
+[project.optional-dependencies]
+jupyter = [
+  "jupyter",
```

Comment:
It doesn't make sense for jupyter to be optional but bokeh mandatory?

Reply:
CNMF and visualization both import bokeh; I think we need it to be present until some future cleanup changes that.

I don't think bokeh pulls jupyter in.

Reply:
Ah right. Though none of the bokeh viz will actually work without jupyter AFAIK.